Release-Notes-R5.md

```diff
@@ -94,25 +93,117 @@ The default values are:
+
+From [#476](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/476), a new environment `regiocloud`
+from a new provider [https://regio-cloud.net/](https://regio-cloud.net/) which now supports SCS is supported.
+
+### ETCD backup and defragmentation
+
+The etcd defragmentation and backup process was enhanced
+[#401](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/401)
+to ensure responsiveness, stability, and safety of the
+Kubernetes (k8s) control plane. The update introduces a script that strategically schedules defragmentation actions
+across etcd nodes within the cluster. The script now intelligently skips defragmentation on non-leader nodes, clusters
+with unhealthy members, or single-member clusters unless forced using optional arguments. The defragmentation sequence
+is optimized by defragmentation of non-leader nodes first, followed by leadership change and defragmentation of the
+selected
+etcd member, and finally, backup and trim operations on the local (ex-leader) etcd member.
+
+### Custom container registry in containerd
+
+The Containerd container runtime interface (CRI) has been enhanced to provide configuration options for interacting
+with public or private container registry hosts. These configuration options are detailed in
+the [Containerd documentation](https://github.com/containerd/containerd/blob/main/docs/hosts.md). This feature enables
+versatile use cases, such as configuring Containerd to utilize a custom CA certificate for interactions with container
+registry hosts or specifying a container registry mirror to bypass restrictions, like DockerHub's pull rate limits. In
+the SCS KaaS reference implementation, users can pass container registry host configuration files to configure
+Containerd cluster-wide. The `containerd_registry_files` Terraform variable allows users to define additional registry
+host configuration files and associated certificates, which will be copied to specific directories on each workload
+cluster node.
+
+The default configuration uses the `registry.scs.community` container registry as a public mirror of
+DockerHub, addressing potential issues related to DockerHub's pull rate limits. This improvement helps ensure seamless
+and efficient container registry interactions within SCS KaaS clusters. The default host config file used can be
+found [here](/terraform/files/containerd/docker.io).
+
+The feature has been implemented in [#432](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/432).
+With this followup PRs:
+
+- [#447](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/447)
+  Add optional containerd registry config files
+- [#452](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/452)
+  Fix default cluster directory ~/cluster-defaults cp command
+- [#472](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/472)
+  Fix configure_containerd.sh script idempotency
```

Comment:
I'm thinking about not mentioning the above two fixes as we do not mention fixes for other release features.